docs/indepth/installation.rst

```diff
@@ -403,3 +403,35 @@ You have also ensured that the ``flocker-deploy`` command line tool is able to c
 Next you may want to perform the steps in :doc:`the tutorial <./tutorial/moving-applications>` , to ensure that your nodes are correctly configured.
 Replace the IP addresses in the ``deployment.yaml`` files with the IP address of your own nodes.
 Keep in mind that the tutorial was designed with local virtual machines in mind, and results in an insecure environment.
+
+
+Experiment on a remote testing cluster
```

Comment:
Might be best to make it clear from the title how this differs from the above - one might spin up a Rackspace / CentOS 7 machine manually in order to experiment.

Also, the installation page is getting awfully long right now so maybe we should ask @alicesowerby about where this should go.